eng/Generate.ps1

```diff
@@ -17,28 +17,45 @@ function Invoke-Block([scriptblock]$cmd) {
     }
 }
 
-$repoRoot = Resolve-Path "$PSScriptRoot\.."
-$testServerTestProject = "$repoRoot\test\AutoRest.TestServer.Tests"
-$testConfiguration = "$testServerTestProject\readme.md"
-$testServerSwaggerPath = "$repoRoot\node_modules\@microsoft.azure\autorest.testserver\swagger"
-$paths = 'url', 'body-string', 'body-complex', 'custom-baseUrl', 'custom-baseUrl-more-options'
-if ($name)
-{
-    $paths = $name
+function Invoke-AutoRest($debugFlags, $testConfiguration, $outputFolder, $inputFile, $name, $namespace, $repoRoot) {
+    Invoke-Block {
+        $outputFlag = if($outputFolder) { "--output-folder=$outputFolder" } else { '' }
```

Comment:
Can we just forward all the non-matched argumetns?

Reply:
I'll have this cleaned up in the next PR. This PR simply allows the check to run. I'll make everything run the same way in the next one.